CMakeLists.txt

```diff
@@ -178,17 +194,13 @@ endif()
 # minimal dependency versions. they are defined here in a single place so
 # they can be easily upgraded, although they might not be used if the
 # dependency is included via `add_subdirectory(...)`.
-set(_acts_actsvg_version 0.4.40)
 set(_acts_autodiff_version 0.6)
-set(_acts_boost_version 1.71.0)
 set(_acts_dd4hep_version 1.21)
 set(_acts_edm4hep_version 0.7)
-set(_acts_geomodel_version 4.6.0)
+set(_acts_eigen3_version 3.3.7)
 set(_acts_podio_version 0.16)
 set(_acts_doxygen_version 1.9.4)
-set(_acts_eigen3_version 3.3.7)
 set(_acts_hepmc3_version 3.2.1)
-set(_acts_nlohmanjson_version 3.2.0)
```

Comment:
isn't this still in use?

same for the others above

Reply:
I don't think its used anymore since it's not set in the external sources cmake file:

set( ACTS_NLOHMANNJSON_SOURCE
   "URL;https://github.com/nlohmann/json/archive/refs/tags/v3.10.5.tar.gz;URL_HASH;SHA1=8969f5ad1a422e01f040ff48dcae9c0e6ad0811d" CACHE STRING "Source to take nlohmann_json from")

Reply:
I've reverted the changes